Standards/scs-XXXX-v1-volume-backup-service.md

```diff
@@ -0,0 +1,94 @@
+As an option, this standard could keep the actual integration of the volume backup feature optional and guide users how to use images as backup targets instead in case the feature is unavailable.
+
+However, it is not guaranteed that the Glance backend storage is separate from the volume storage.
+For instance, both could be using the same Ceph cluster.
+In such case, the images would not count as genuine backups.
+
+Although users are able to download images and transfer them to a different storage location, this approach might also prove unfeasible depending on the image size and the existence (or lack) of appropriate target storage on the user side.
+
+Furthermore, incremental backups are not possible when creating Glance images from volumes either.
+This results in time-consuming backup operations of fully copying a volume everytime a backup is created.
+
+#### Focus on feature availability, make feature mandatory
+
+This option is pretty straightforward.
+It would make the volume backup feature mandatory for SCS clouds.
+This way users can expect the feature to be available and usable.
+
+With this, users can leverage functionalities like incremental backups and benefit from optimized performance of the backup process due to the tight integration with the volume service.
+
+However, it does not seem feasible to also mandate having a separate storage backend for volume backups at the same time due to potential infrastructure limitations at CSP-side making it hard or even impossible to offer.
+As such, the actual benefit of backups in terms of reliability and security aspects would be questionable if a separate storage backend is not mandated and therefore not guaranteed.
+
+This approach would focus on feature availability rather than backup reliability.
+
+#### Focus on backup reliability, make separate backend mandatory
+
+As an alternative, the volume backup feature availability could be made optional but in case a CSP chooses to offer it, the standard would mandate a separate storage backend to be used for volume backups.
+This way, failures of the volume storage backend would not directly impact the availability and safety of volume backups, making them actually live up to their name.
+
+In contrast to the above, this approach would focus on backup reliability rather than feature availability.
+
+## Standard
+
+This standard decides to go with the second option and makes the volume backup feature mandatory in the following way:
+
+In an SCS cloud, the volume backup functionality MUST be configured properly and its API as defined per `/v3/{project_id}/backups` MUST be offered to customers.
+If using Cinder, a suitable [backup driver](https://docs.openstack.org/cinder/latest/configuration/block-storage/backup-drivers.html) MUST be set up.
+
+The volume backup target storage SHOULD be a separate storage system from the one used for volumes themselves.
```

Comment:
As it is not possible to know as a user, whether a volume backup storage is different from the normal volume storage. We should encourage CSPs to give that information to customers. This could also be done via gaiax credentials, couldn't it? Maybe we need another issue for this part.

Reply:
This would require a deep dive into the [Gaia-X Ontology](https://gaia-x.gitlab.io/gaia-x-community/gaia-x-self-descriptions/service/service.html) to figure out whether this can be expressed using Gaia-X Credentials (formerly Self-Descriptions) appropriately.

As long as we don't have a standardized and proven way of representing arbitrary self-description information about an SCS cloud and its services in Gaia-X Credentials, I'm hesitant to add any such suggestions to the standard yet.

Reply:
Yeah, I thought so - Maybe we can discuss in the next Standardization SIG meeting or tomorrow, if and how we can use the gaia-x credentials for tests.